test/exhal_test.exs

```diff
@@ -239,16 +271,16 @@ defmodule ExHalTest do
       end
     end
 
-    defp doc do
+    defp doc(scheme_and_domain \\ "http://example.com/") do
```

Comment:
Maybe `base_url` would better convey how this argument is used?

Reply:
done